ccc-js/src/main/javascript/package-res/ccc/plugin/box/box-plot-panel.js

```diff
@@ -229,11 +229,11 @@ def
     _buildSceneCore: function(data, axisSeriesDatas, axisCategDatas) {
         //  chart measureVisualRoles would only return bound visual roles.
         var measureVisualRoleInfos = def.query(this.visualRoleList)
-                .where(function(r) { return r.isMeasureEffective; })
+                .where(function(r) { return !r.isDiscrete() && r.isMeasure; })
                 .select(function(r) {
                     return {
                         roleName: r.name,
-                        dimName:  r.grouping.singleDimensionName
+                        dimName:  r.lastDimensionName()
```

Comment:
Actually, I think this line was ok, even though the results are similar.

Reply:
Nope, tested, otherwise you will get an exception on console and the chart will not be displayed

Reply:
Ohh, I see. Then, please, use `r.grouping && r.grouping.singleDimensionName` instead.